product-service/functions/getProducts.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+const db = new AWS.DynamoDB.DocumentClient();
+
+const productTable = process.env.PRODUCT_TABLE;
+const stockTable = process.env.STOCK_TABLE;
+
+module.exports.handler = async (event) => {
+  const products = await db
```

Comment:
Everytime you use Dynamo db avoid as much as possible doing scans. Since these consume a lot of compute, and cause process break when the table is too big. In this case the most appropiate thing to do is a query intead.